### ISSUE TITLE
Add CLI options to change Descriptor type

### ISSUE THREAD

**Issue by AstromechZA** [AstromechZA/bunkr#26]
This functionality needs to match the functionality provided by the Archive Security Window. 

idea: change the changepassword command to a change-security command. 

Use the descriptor IDENTIFIER as a subcommand name. 

eg:
```
bunkr change-security <archive.bunkr> plaintext

bunkr change-security <archive.bunkr> pbkdf2 <password-file> [options]

bunkr change-security <archive.bunkr> scrypt <password-file> [options]
```

**Comment by AstromechZA** [comment]
Adding all of these parameters to the `create` command would be a pain.
I'm thinking we should instead change the user flow into:

```
# First create a brand new archive WITH NO INVENTORY ENCRYPTION
bunkr create <archive.bunkr> [--no-encryption] [--no-compression]

# Then set up the security
bunkr change-security <archive.bunkr> [password file 1] scrypt [password-file 2] [scrypt options]
```

**Comment by AstromechZA** [comment]
Fixed in 033a489
